composer.json

```diff
@@ -98,6 +98,7 @@
         "twig/twig": "^1.41 || ^2.0"
     },
     "require-dev": {
+        "contao/imagine-svg": "^0.2.2",
```

Comment:
I guess that's not just a dev requirement, or is it?

Reply:
Resp. at least it should also be listed as suggestion, if we don't want to install it by default. But is there any reason you would not want to install it? Is that somehow super heavy to initialize or has another negative impact on performance?

Reply:
will add it to suggestions